events/dvg_atlantia.txt

```diff
@@ -0,0 +1,795 @@
+			}
+		set_global_variable = nouvellsatian_support
+	
+	ai_chance = {
+			base = 20
+		}
+       }
+}
+
+dvg_atlantia.6 = {
+	type = country_event
+	placement = ROOT
+	event_image = {
+		video = "gfx/event_pictures/unspecific_politicians_arguing.bk2"
+	}
+
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "texan_war_of_independence.1"
+
+	icon = "gfx/interface/icons/event_icons/event_portrait.dds"
+
+	title = dvg_atlantia.6.t
+	desc = dvg_atlantia.6.d
+	flavor = dvg_atlantia.6.f
+
+	duration = 1
+
+	trigger = {
+	}
+
+	immediate = {
+		c:ATL = {
+			
+			
+		}
+	}
+
+	option = {
+		name = dvg_atlantia.6.a
+		default_option = yes
+		
+		change_relations = {
+				country = c:NEA
+				value = -15
+			}
+	}
+	
+	
+}
+
+dvg_atlantia.7 = {
+	type = country_event
+	placement = ROOT
+	event_image = {
+		video = "gfx/event_pictures/southamerica_election.bk2"
+	}
+
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "texan_war_of_independence.1"
+
+	icon = "gfx/interface/icons/event_icons/event_portrait.dds"
+
+	title = dvg_atlantia.7.t
+	desc = dvg_atlantia.7.d
+	flavor = dvg_atlantia.7.f
+
+	duration = 1
+
+	trigger = {
+	}
+
+	immediate = {
+		c:ATL = {
+			
+			
+		}
+	}
+
+	option = {
+		name = dvg_atlantia.7.a
+		default_option = yes
+		create_diplomatic_pact = {
+			country = c:NEA
+			type = puppet
+		}
+		
+		
+	}
+	
+	
+}
+
+dvg_atlantia.8 = {
+	type = country_event
+	placement = ROOT
+	event_image = {
+		video = "gfx/event_pictures/southamerica_election.bk2"
+	}
+
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "texan_war_of_independence.1"
+
+	icon = "gfx/interface/icons/event_icons/event_portrait.dds"
+
+	title = dvg_atlantia.8.t
+	desc = dvg_atlantia.8.d
+	flavor = dvg_atlantia.8.f
+
+	duration = 1
+
+	trigger = {
+	}
+
+	immediate = {
+		
+	}
+
+	option = {
+		name = dvg_atlantia.8.a
+		default_option = yes
+		create_diplomatic_pact = {
+			country = c:ATL
+			type = puppet
+		}
+		
+		
+	}
+	
+	
+}
+
+dvg_atlantia.9 = {
+	type = country_event
+	placement = ROOT
+	event_image = {
+		video = "gfx/event_pictures/unspecific_politicians_arguing.bk2"
+	}
+
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "event:/SFX/Events/middleeast/event:/SFX/Events/europenorthamerica/before_the_battle"
+
+	icon = "gfx/interface/icons/event_icons/event_portrait.dds"
+
+	title = dvg_atlantia.9.t
+	desc = dvg_atlantia.9.d
+	flavor = dvg_atlantia.9.f
+
+	duration = 2
+
+	trigger = {
+		
+		
+		
+	}
+
+	immediate = {
+		c:AMA = {
+			add_modifier = {
+			name = dvg_demoratic_miltias
+			days = 1825
+			is_decaying	= yes
+		}
+			
+		}
+		
+	}
+
+	option = {
+		name = dvg_atlantia.9.a
+		default_option = yes
+		
+		c:LOT = {
+		 create_diplomatic_play = {
+			 name = reconquering_atlantia.1.t
+		
+			target_state = s:STATE_COTE_DU_CAFE.region_state:ATL
+			
+			war = no
+			
+			 type = dp_return_state

```

Comment:
Shouldent it be annex country? 
![image](https://user-images.githubusercontent.com/116951759/230716675-6a280245-2b81-48d7-8c68-4e74a633bd9f.png)